staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -2815,13 +2815,6 @@ func TestWatchStreamSeparation(t *testing.T) {
 			expectBookmarkOnEtcd:       true,
 			expectBookmarkOnWatchCache: true,
 		},
-		{
```

Comment:
This test case no longer can be tested, however that's ok because it doesn't happen.